styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,1041 @@
+      background-position: 0 -48px !important; }
+
+      .icon-menu-mobile {
+        @svg: escape('<svg width="56" height="812" viewBox="0 0 56 812" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.25 3h-2.5A3.25 3.25 0 0 0 9.5 6.25v2.5c0 .644.188 1.245.51 1.75H9A3.5 3.5 0 0 0 5.5 14v3A2.5 2.5 0 0 0 8 19.5h1v2a2 2 0 0 0 2 2h2a1 1 0 0 0 1-1 1 1 0 0 0 1 1h2a2 2 0 0 0 2-2v-2h1a2.5 2.5 0 0 0 2.5-2.5v-3a3.5 3.5 0 0 0-3.5-3.5h-1.01c.323-.505.51-1.106.51-1.75v-2.5A3.25 3.25 0 0 0 15.25 3m0 7.5A1.75 1.75 0 0 0 17 8.75v-2.5a1.75 1.75 0 0 0-1.75-1.75h-2.5A1.75 1.75 0 0 0 11 6.25v2.5c0 .966.784 1.75 1.75 1.75zm-2.5 1.5H9a2 2 0 0 0-2 2v3a1 1 0 0 0 1 1h1v-3.5a.5.5 0 0 1 1 0V21a1 1 0 0 0 1 1h2v-3a1 1 0 1 1 2 0v3h2a1 1 0 0 0 1-1v-6.5a.5.5 0 0 1 1 0V18h1a1 1 0 0 0 1-1v-3a2 2 0 0 0-2-2zm-8.227 9.273c.622-.37 1.498-.656 2.484-.887v.658c-.506.142-.95.309-1.29.501-2.663 1.364.183 3.273 8.263 3.273s10.926-1.909 8.264-3.273c-.324-.183-.74-.343-1.214-.48v-.726c.925.247 1.745.545 2.407.843C27.11 23.09 24.447 26 13.98 26S.942 23.09 4.523 21.273M10 34a3 3 0 0 1 3-3h2a3 3 0 0 1 3 3v2a3 3 0 0 1-3 3h4a3 3 0 0 1 3 3v3a2 2 0 0 1-2 2h-1v-4.5a.5.5 0 0 0-1 0V50a1 1 0 0 1-1 1h-1a1 1 0 0 1-1-1v-3a1 1 0 1 0-2 0v3a1 1 0 0 1-1 1h-1a1 1 0 0 1-1-1v-7.5a.5.5 0 0 0-1 0V47H8a2 2 0 0 1-2-2v-3a3 3 0 0 1 3-3h4a3 3 0 0 1-3-3zM4.523 49.273c.62-.37 1.494-.655 2.477-.886v.659c-.503.141-.944.307-1.284.5-2.662 1.363.184 3.272 8.264 3.272s10.926-1.909 8.264-3.273c-.33-.186-.758-.35-1.244-.488v-.726c.938.249 1.768.55 2.437.85C27.11 51.092 24.447 54 13.98 54S.942 51.09 4.523 49.273M22 61.5H6a.5.5 0 0 0-.5.5v16.5H14V76a6 6 0 0 1 6-6h2.5v-8a.5.5 0 0 0-.5-.5M14 80h2l8-8V62a2 2 0 0 0-2-2H6a2 2 0 0 0-2 2v18zm1.5-1.621 6.879-6.879H20a4.5 4.5 0 0 0-4.5 4.5zM9 64.5h10a.5.5 0 0 1 .5.5v1.5a.5.5 0 0 1-.5.5H9a.5.5 0 0 1-.5-.5V65a.5.5 0 0 1 .5-.5M7 65a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v1.5a2 2 0 0 1-2 2H9a2 2 0 0 1-2-2zm0 5.75a.75.75 0 0 1 .75-.75h4a.75.75 0 0 1 0 1.5h-4a.75.75 0 0 1-.75-.75M7.75 73a.75.75 0 0 0 0 1.5h4a.75.75 0 0 0 0-1.5zM10 89.5h8a4.5 4.5 0 0 1 4.5 4.5v8a4.5 4.5 0 0 1-4.5 4.5H5.5V94a4.5 4.5 0 0 1 4.5-4.5M4 94a6 6 0 0 1 6-6h8a6 6 0 0 1 6 6v8a6 6 0 0 1-6 6H4zm5.75-1a.75.75 0 0 0 0 1.5h8.5a.75.75 0 0 0 0-1.5zm0 4a.75.75 0 0 0 0 1.5h8.5a.75.75 0 0 0 0-1.5zM9 101.75a.75.75 0 0 1 .75-.75h4.333a.75.75 0 0 1 0 1.5H9.75a.75.75 0 0 1-.75-.75M10 116a6 6 0 0 0-6 6v14h14a6 6 0 0 0 6-6v-8a6 6 0 0 0-6-6zm-.25 5a.75.75 0 0 0 0 1.5h8.5a.75.75 0 0 0 0-1.5zm0 4a.75.75 0 0 0 0 1.5h8.5a.75.75 0 0 0 0-1.5zM9 129.75a.75.75 0 0 1 .75-.75h4.333a.75.75 0 0 1 0 1.5H9.75a.75.75 0 0 1-.75-.75m1.25 13.75h2.5a3.25 3.25 0 0 1 3.25 3.25v2.5q0 .346-.07.672a8 8 0 0 0-3.675 2.578H6.5a2 2 0 0 0-2 2v3a1 1 0 0 0 1 1H7V155a.5.5 0 0 1 1 0v6.5a1 1 0 0 0 1 1h2v-2.21c.25.672.588 1.302 1 1.875V163a1 1 0 0 1-1 1H9a2 2 0 0 1-2-2v-2H5.5a2.5 2.5 0 0 1-2.5-2.5v-3a3.5 3.5 0 0 1 3.5-3.5h1.01a3.24 3.24 0 0 1-.51-1.75v-2.5a3.25 3.25 0 0 1 3.25-3.25m4.25 5.75a1.75 1.75 0 0 1
```

Comment:
This is a super bright border color for these!

Reply:
![CleanShot 2024-09-20 at 07 03 00](https://github.com/user-attachments/assets/52901704-cd64-41d3-bfae-0afcc9cfbcfc)

Looks weird with this hover state.